cpp/EncryptionMaterial.hpp

```diff
@@ -0,0 +1,49 @@
+//
+// Created by hyu on 2/5/18.
```

Comment:
Can you please remove your name from the code?
We don't need to keep somebody's name in the code. Instead, those change history should be in the git history.

Reply:
Also add copyright here.